test/unittests/t_catalog.cc

```diff
@@ -23,7 +23,7 @@ class T_Catalog : public ::testing::Test {
   virtual void SetUp() {
     catalog = NULL;
     nested = NULL;
-    sandbox = CreateTempDir("./cvmfs_ut_catalog");
+    sandbox = CreateTempDir("/tmp/cvmfs_ut_catalog");
```

Comment:
Totally true... I wasn't aware of that, so I thought I modified that by mistake... I will fix it